OpenDreamRuntime/DreamManager.Connections.cs

```diff
@@ -358,3 +358,24 @@ public void Execute(IConsoleShell shell, string argStr, string[] args) {
         dreamManager.HotReloadResource(args[0]);
     }
 }
+
+
+public sealed class ForceGarbageCollectionCommand : IConsoleCommand {
```

Comment:
This is already a server command provided by RT, you can run `gc` in the server's console.

Reply:
oh no worries, I'll delete